Provisioning: Allow saving of provisioned dashboards (#19820)
grafana/grafana: docs/sources/administration/provisioning.md

#### Making changes to a provisioned dashboard

> It's possible to make changes to a provisioned dashboard in Grafana UI, but there's currently no possibility to automatically save the changes back to the provisioning source. 

Reviewer: ```suggestion
It's possible to make changes to a provisioned dashboard in the Grafana UI. However, is not possible to automatically save the changes back to the provisioning source. 
```

Reviewer 2: However, **it** is not possible...

> If `allowUiUpdates` is configured to `true` and you make changes to a provisioned dashboard you can `Save` the dashboard and the changes will be persisted to the Grafana database.

Reviewer: ```suggestion
If `allowUiUpdates` is set to `true` and you make changes to a provisioned dashboard, you can `Save` the dashboard then the changes will be persisted to the Grafana database.
```

> **Note.** 

> > If a provisioned dashboard is saved from the UI and then later updated on disk the dashboard stored in the database will always be overwritten. The `version` property in the json file will not affect this even if it is lower then the existing dashboard.

Reviewer: ```suggestion
> If a provisioned dashboard is saved from the UI and then later updated from the source, the dashboard stored in the database will always be overwritten. The `version` property in the JSON file will not affect this, even if it is lower than the existing dashboard.
```

> 

> > If a provisioned dashboard is saved from the UI and then later the file is removed from disk, the dashboard stored in the database will be deleted unless the configuration option `disableDeletion` is set to true.

Reviewer: ```suggestion
> If a provisioned dashboard is saved from the UI and the source is removed, the dashboard stored in the database will be deleted unless the configuration option `disableDeletion` is set to true.
```

> If `allowUiUpdates` is configured to `false` and you make changes to a provisioned dashboard you can `Save` the dashboard which will bring up a *Cannot save provisioned dashboard* dialog like seen in the screenshot below.

Reviewer: ```suggestion
If `allowUiUpdates` is configured to `false`, you are not able to make changes to a provisioned dashboard. When you click `Save`, Grafana brings up a *Cannot save provisioned dashboard* dialog. The screenshot below illustrates this behavior.
```

> Here available options will let you `Copy JSON to Clipboard` and/or `Save JSON to file`. This can help you synchronize your dashboard changes back to the provisioning source.

Reviewer: ```suggestion
Grafana offers options to export the JSON definition of a dashboard. Either `Copy JSON to Clipboard` or `Save JSON to file` can help you synchronize your dashboard changes back to the provisioning source.
```

> Note: The JSON shown in input field and when using `Copy JSON to Clipboard` and/or `Save JSON to file` will have the `id` field automatically removed to aid the provisioning workflow.

Reviewer: ```suggestion
Note: The JSON definition in the input field when using `Copy JSON to Clipboard` or `Save JSON to file` will have the `id` field automatically removed to aid the provisioning workflow.
```
